.github/actions/release-package/index.mjs

```diff
@@ -5,9 +5,14 @@ import { existsSync, readFileSync, writeFileSync } from 'fs';
 const inputs = {
   path: process.env.INPUT_PATH,
   suffix: process.env.INPUT_SUFFIX,
-  publishPackages: process.env.PUBLISH_PACKAGES?.split(',').map((pkg) => pkg.trim()),
+  publishPackages: process.env.PUBLISH_PACKAGES
+    ? process.env.PUBLISH_PACKAGES.split(',').map((pkg) => pkg.trim())
+    : null,
 };
 
+console.log('Inputs:');
+console.log(JSON.stringify(inputs, null, 2));
```

Comment:
The logging here is left intentionally. It is useful for future investigations how github workflow arguments are propagated